lib/PINNBench/Burgers/benchmark.py

```diff
@@ -11,5 +10,8 @@ class PINNBurgersBenchmark(Benchmark):
     version = "0.0.1"
 
     requires = {
-        "py-pip-requirements": {"type": "pip", "name": "-r "+ os.path.join(DIR, "requirements.txt")}, # not sure what this line should do or I should do to match this line.
+        "py-pip-requirements": {
+            "type": "pip",
+            "name": "-r " + os.path.join(DIR, "requirements.txt"),
+        },  # not sure what this line should do or I should do to match this line.
```

Comment:
`# not sure what this line should do or I should do to match this line.` maybe this line can be removed